### DISCUSSION TITLE
Admin console doesn't show anywhere even if logging in with an admin user.

### DISCUSSION THREAD

**Discussion by GeniusDev0** [sanidhyy/duolingo-clone#14] [General]
Title.

**Comment by sanidhyy** [comment]
Please ensure there are no spaces in the `.env` file when setting the admin user IDs. Also, verify that the Clerk ID matches exactly with the one in the Clerk dashboard. Clear the Next.js cache, restart the app, and confirm that there are no typos. Make sure the environment variable name is written exactly as `CLERK_ADMIN_IDS`.
